src/hud.h

```diff
@@ -175,7 +175,11 @@ class Hud {
 	v3f m_selected_face_normal;
 
 	video::SMaterial m_selection_material;
-	bool m_use_selection_mesh;
+	
+	enum { 
```

Comment:
I'm going to take that to mean this change isn't needed for now. Let me know if it is.